polygon-digital-carbon/src/utils/CarbonProject.ts

```diff
@@ -48,6 +49,19 @@ export function loadOrCreateCarbonProject(registry: string, projectID: string):
       }
     }
 
+    if (registry == 'CMARK') {
+      for (let i = 0; i < CMARK_PROJECT_INFO.length; i++) {
+        let projectInfo = CMARK_PROJECT_INFO[i]
+        if (projectID == projectInfo[0]) {
+          project.name = projectInfo[1]
+          project.methodologies = projectInfo[2]
+          project.category = projectInfo[3]
+          project.country = projectInfo[4]
+          project.region = projectInfo[5]
```

Comment:
does country need to be the two digit country code...? I can't remember explicitly so this might not be the case; but wanted to highlight in case that is necessary for the api

Reply:
Honestly, I am not even sure we have to do this. The API is going to fetch the data from the CMS anyways. The use case would be for other direct consumers of the subgraph...

Reply:
Hmm... Looks like I cannot. It was possible for moss because it is the only handler called I suppose.